bin/start

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
For the sake of following a more Rails-ish convention, maybe we should use a `Procfile` with foreman instead of a custom bash script for this. We can create a `Procfile.dev` and a [`.foreman`](https://ddollar.github.io/foreman/#DEFAULT-OPTIONS) file to point to this file as the default to use locally without confusing Heroku with local-development only stuff in a generic`Procfile`.

Reply:
Thanks for this, I've learnt something new today 👍 I've amended the commit with the change